src/components/Button.js

```diff
@@ -41,16 +49,41 @@ const Secondary = styled(StyledButton)`
   }
 `
 
-const Button = ({ to, isSecondary, children, className }) => {
+const Button = ({
+  to,
+  isSecondary,
+  children,
+  className,
+  marginLeft = 0,
```

Comment:
Not sure this is a pattern we want to start rolling out on components 🤔 

Note to self to find a better approach.

Reply:
I gotta say I love the flexibility of being able to affect positioning of components without having to create a `StyledButton` but if there's a better way, I'm in!